src/lib/log.c

```diff
@@ -35,6 +35,12 @@
 static void log_internal(FILE *out, const char *file, int line, const char *label, const char *format, va_list args_list);
 static int cur_log_level = LOG_INFO;
 static log_fn_t log_function = log_internal;
+static bool mute = false;
+
+void log_mute(bool state)
```

Comment:
I don't like the idea on supporting a log mute. This looks like a problem on how the log is called and that the function that is using it could be improve to print log correctly.